docs/docs.md

```diff
@@ -13,11 +13,14 @@ from dynamics import DynamicsClient
 - client_id: str - Client id (e.g. UUID).
 - client_secret: str - Client secret (e.g. OAuth password).
 - scope: List[str] - List of urls in form: 'https://[Organization URI]/scope'. Defines the database records that the API connection has access to.
```

Comment:
Type of `scope` is now also `Optional`. It should also mention that its mutually exclusive with resource.

Also, put the default value as `None `like in [here](https://github.com/MrThearMan/dynamics-client/pull/4/files#diff-05518836594e79b191a867fc78aa2c759b6bc9cbd3565f8d0e858561cee050a2R59). Same for `resource`.

Reply:
We can do this either way; after finding out that `resource=` can complement `scope=`, I went back to the default behaviour previously coded into the client (requiring scope).

If you prefer, I can update the behaviour to require at least one of the two to be set.

Reply:
I think both should be typed as `Optional`, since if you're using `resource`, you don't need to know `scope`, and vice verca. Then documentation should mention that you should try scope by default and try resource if [this](https://github.com/MrThearMan/dynamics-client/pull/4/files#diff-05518836594e79b191a867fc78aa2c759b6bc9cbd3565f8d0e858561cee050a2R22) happens.